buildstockbatch/eagle.py

```diff
@@ -75,13 +82,20 @@ def __init__(self, project_filename):
     def validate_project(cls, project_file):
         super(cls, cls).validate_project(project_file)
         # Eagle specific validation goes here
+        cls.validate_output_directory_eagle(project_file)
+        logger.info("Eagle Validation Successful")
+        return True
+
+    @classmethod
+    def validate_output_directory_eagle(cls, project_file):
+        cfg = get_project_configuration(project_file)
+        output_dir = path_rel_to_file(project_file, cfg['output_directory'])
+        if not (output_dir.startswith('/scratch') or output_dir.startswith('/projects')):
+            raise ValidationError(f"`output_directory` must be in /scratch or /projects, `output_directory` = {output_dir}")
```

Comment:
Added this check because I accidentally set up a run to put my outputs in `/home/nmerket`. I should know better and made this mistake. This is a very common error for new people as well who aren't used to Eagle, so I made it so it won't let you try to save your outputs somewhere that doesn't make sense.